.github/scripts/strings_reader.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
```

Comment:
The script looks good overall, can we also print the path of the file that is being processed?

Reply:
We are already doing that

```python
print("Checking strings file at -> {}".format(strings_file_path))
```

Reply:
Ah, I think I missed that line.